orange-datamodel/src/test/resources/real.orange.json

```diff
@@ -23,8 +23,8 @@
       "pctExcAdapter": 0.0,
       "pctExcMapQ": 0.04985,
       "pctExcDupe": 0.10868,
-      "pctExcUnpaired": 0.00071,
-      "pctExcBaseQ": 0.000040,
+      "pctExcUnpaired": 7.1E-4,
+      "pctExcBaseQ": 4.0E-5,
```

Comment:
See other comment, not entirely sure why this happens, though I imagine it must be some local setting on my laptop that is different from the previous update. I'll leave this as-is since it seems fine.